sources/packages/web/src/components/common/modals/EducationProgramDeactivationModal.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <v-form ref="deactivateProgramForm">
+    <modal-dialog-base
+      :showDialog="showDialog"
+      title="Confirm deactivation"
+      ref="confirmDeactivationModal"
+      okLabel="Confirm deactivation"
+      keepModalOpen="true"
+      ><template #content
+        ><span class="font-bold"
+          >Please confirm you no longer require this program?</span
+        >
+        <p class="mt-5">
+          Confirming deactivation means that this will no longer be visible to
+          students when starting an application. It wil not impact any students
+          with active applications for offerings under this program. This action
+          cannot be undone.
+        </p>
+        <v-textarea
+          v-if="notesRequired"
+          v-model="noteDescription"
+          variant="outlined"
+          label="Note"
+          :rules="[checkNotesLengthRule]"
+          required
+          class="mt-4"
+          hide-details="auto"
+        ></v-textarea>
+      </template>
+      <template #footer>
+        <footer-buttons
+          primaryLabel="Confirm deactivation"
+          primaryButtonColor="danger"
```

Comment:
Was the primary color set to danger here highlighting the part that they are deactivating?

Reply:
Yes, I belive that the intention was to be extra clear about the action impact because there is no way to revert it.